cmake/ngraph.cmake

```diff
@@ -36,10 +35,10 @@ endif(USE_NGRAPH_DISTRIBUTED)
 ExternalProject_Add(
 	ext_ngraph
 	GIT_REPOSITORY https://github.com/NervanaSystems/ngraph.git
-	GIT_TAG v0.6.0
+	GIT_TAG "v0.6.0"
```

Comment:
Cmake on Mac peculiarity?

Reply:
No, a leftover from a change I'd experimented with, coupled with my tendency for excessive caution in languages like Bash and CMake.  Can remote if you want.